lib/pages/login_page.dart

```diff
@@ -23,7 +23,9 @@ class _LoginPageState extends State<LoginPage> {
             decoration: BoxDecoration(
               image: DecorationImage(
                   fit: BoxFit.cover,
-                  image: AssetImage('assets/images/sign_in_background.png')),
+                  image: AssetImage('assets/images/sign_in_background.png'),
+                  colorFilter: ColorFilter.mode(
+                      Color.fromRGBO(0, 0, 0, 0.4), BlendMode.overlay)),
```

Comment:
optional: you can make it more readable with `Colors.black.withOpacity(0.4)` 💯

Reply:
thanks, updated